nodejs/package.json

```diff
@@ -18,15 +18,15 @@
         "test": "mocha test/*.js",
         "buildProto": "pbjs -t static-module -w commonjs -o gtfs-realtime.js ../gtfs-realtime.proto"
     },
-    "engines": [
-        "node >= 0.10.0"
-    ],
+    "engines": {
+        "node": "18.12.1"
```

Comment:
Note that this is a breaking change!

By definition, any change to the current version `0.0.5` is considered a breaking change by SemVer:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

I'd still advocate for picking e.g. `0.1.0` or `1.0.0` to make this especially clear.